src/server.c

```diff
@@ -4839,16 +4838,29 @@ sds genRedisInfoString(const char *section) {
     /* CPU */
     if (allsections || defsections || !strcasecmp(section,"cpu")) {
         if (sections++) info = sdscat(info,"\r\n");
-        info = sdscatprintf(info,
+
+        struct rusage self_ru, c_ru;
+        getrusage(RUSAGE_SELF, &self_ru);
+        getrusage(RUSAGE_CHILDREN, &c_ru);
+        info = sdscatfmt(info,
         "# CPU\r\n"
-        "used_cpu_sys:%ld.%06ld\r\n"
-        "used_cpu_user:%ld.%06ld\r\n"
-        "used_cpu_sys_children:%ld.%06ld\r\n"
-        "used_cpu_user_children:%ld.%06ld\r\n",
+        "used_cpu_sys:%I.%I\r\n"
+        "used_cpu_user:%I.%I\r\n"
+        "used_cpu_sys_children:%I.%I\r\n"
+        "used_cpu_user_children:%I.%I\r\n",
```

Comment:
reverting your change.
```suggestion
        "used_cpu_sys:%ld.%06ld\r\n"
        "used_cpu_user:%ld.%06ld\r\n"
        "used_cpu_sys_children:%ld.%06ld\r\n"
        "used_cpu_user_children:%ld.%06ld\r\n",
```